tests/testthat/test-model-frame.R

```diff
@@ -6,3 +6,33 @@ test_that("get_age_group_id_out() returns expected groups", {
   expect_equal(get_age_group_id_out(1:17), 1:29)
   expect_equal(get_age_group_id_out(4:17), c(4:21, 25:29))
 })
+
+
+areas <- create_areas(mwi_area_levels, mwi_area_hierarchy, mwi_area_boundaries)
```

Comment:
Can you move lines 11-20 inclusive into a helper file instead? `tests/testthat/helper-model-frame.R` it gets evaluated before the tests so `areas`, `spec`, `naomi_mf` will be available to use within the test code as you have here.

Reply:
commit 46f5be6